dotcom-rendering/src/components/ExpandableMarketingCardWrapper.importable.tsx

```diff
@@ -0,0 +1,82 @@
+import { getCookie } from '@guardian/libs';
+import { useEffect, useState } from 'react';
+import type { DailyArticle } from '../lib/dailyArticleCount';
+import { getDailyArticleCount } from '../lib/dailyArticleCount';
+import { getLocaleCode } from '../lib/getCountryCode';
+import { useAB } from '../lib/useAB';
+import { ExpandableMarketingCard } from './ExpandableMarketingCard';
+
+interface Props {
+	guardianBaseURL: string;
+}
+
+const isFirstArticle = () => {
+	const [dailyCount = {} as DailyArticle] = getDailyArticleCount() ?? [];
```

Comment:
This call to `getDailyArticleCount()` uses local storage. At this point, we already know that the user is in the US.